code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -283,33 +283,33 @@
 	var/mob/living/carbon/victim = M
 	if(method == TOUCH || method == VAPOR)
 		//check for protection
-		var/mouth_covered = victim.is_mouth_covered()
 		var/eyes_covered = victim.is_eyes_covered()
```

Comment:
is_eyes_covered returns whatever thing covers the eyes first, including gas masks.

Reply:
What would be a better thing? Checking if there is an item in their eye slot?